scripts/hid_configurator/configurator.py

```diff
@@ -78,7 +79,12 @@ def create_report(recipient, event_id, event_data):
     else:
         event_data_len = 0
 
-    report = struct.pack('<BHBB', REPORT_ID, recipient, event_id, event_data_len)
+    if is_fetch:
```

Comment:
we don't need data if it is a fetch or do we?

Reply:
```
>>> struct.pack('')
b''
>>> bool(struct.pack(''))
False
```
But if you prefer to set `event_data = None` I can change it.

Reply:
but we do not check if data is empty
either assert this is empty or do not process it.

I think it may be better to create two methods: create_set_report and create_fetch_report

Reply:
done